jekyll/_cci2/github-integration.adoc

```diff
@@ -102,7 +102,7 @@ Once disconnected, you will be redirected to the CircleCI login page. To reconne
 
 **What is a deploy key?**
 
-When you add a new project, CircleCI creates a deployment key on GitHub for your project. A deploy key is an SSH key-pair, one public, one private. GitHub stores the public key, and CircleCI stores the private key. The deployment key gives CircleCI access to a single repository. To prevent CircleCI from pushing to your repository, this deployment key is read-only.
+When you add a new project, CircleCI creates a deployment key on GitHub for your project. A deploy key is an SSH key-pair, one public, one private. GitHub stores the public key, and CircleCI stores the private key. The deployment key gives CircleCI access to a single repository. To prevent CircleCI from pushing to your repository, this deployment key is read-only. To set up a CircleCI project, you must  link:https://support.circleci.com/hc/en-us/articles/36222961831067-Unable-to-Set-Up-New-Projects-in-New-GitHub-Organizations/[enable deploy keys in your GitHub Organization Settings].
```

Comment:
```suggestion
When you add a new project, CircleCI creates a deployment key on GitHub for your project. A deploy key is an SSH key-pair, one public, one private. GitHub stores the public key, and CircleCI stores the private key. The deployment key gives CircleCI access to a single repository. To prevent CircleCI from pushing to your repository, this deployment key is read-only. To set up a CircleCI project, you must first enable the creation of deploy keys in your GitHub organization. For more information, see link:https://support.circleci.com/hc/en-us/articles/36222961831067-Unable-to-Set-Up-New-Projects-in-New-GitHub-Organizations/[this support article].
```

Reply:
Is this a problem that comes up a lot @HennaAbbas ? If so we should add a troubleshooting section to the create a project guide too: https://circleci.com/docs/create-project/#set-up-a-project